### PR TITLE
CNV#30908: guest log access

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.15+

Issue: [CNV-30908](https://issues.redhat.com//browse/CNV-30908)

Link to docs preview:
- [Guest system logs](https://71868--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting#guest-system-logs_virt-troubleshooting) (new section)
- [Viewing virt-launcher pods with the web console](https://71868--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting#virt-viewing-virt-launcher-pod-logs-web_virt-troubleshooting) (extra update)

QE review:
- [x] QE has approved this change.

Additional information:
- I also updated an existing log procedure for accuracy, which included renaming it (modules/virt-viewing-virtual-machine-logs-web.adoc -> modules/virt-viewing-virt-launcher-pod-logs-web.adoc)
- I reorganized this page somewhat, to make sense of my new section (hopefully)
- Note to reviewers: at this point, this page probably needs to be split into multiple assemblies and made more modular, but that work is out of the scope of my Jira

### CODE REVIEW

File: modules/virt-enable-guest-log-default-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/support/virt-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enable-guest-log-default-cli_{context}"]
+= Enabling default access to VM guest system logs with the CLI
+
+You can enable default access to virtual machine (VM) guest system logs by editing the `HyperConverged` custom resource (CR).
```
Comment (gaurav-nelson) [inline modules/virt-enable-guest-log-default-cli.adoc:9]: **thought:** I see that both VM and CR are already use on the page before this section, I'm not sure if we still want to define it here. Based on other modules I think we are doing this in all modules.

File: virt/support/virt-troubleshooting.adoc [hunk 2]
```diff
@@ -37,42 +37,56 @@ You can view resource events by running the following command:
...
-You can configure the verbosity level of the logs by editing the `HyperConverged` custom resource (CR).
+Viewing the boot logs of VM guests can help diagnose issues. You can configure access to guests' logs and view them by using either the {product-title} web console or the `oc` CLI. 
 
-include::modules/virt-viewing-logs-cli.adoc[leveloffset=+3]
+This feature is disabled by default. If a VM does not explicitly have this setting enabled or disabled, it inherits the cluster-wide default setting.
 
-include::modules/virt-configuring-pod-log-verbosity.adoc[leveloffset=+3]
+[IMPORTANT]
+====
+If sensitive information such as credentials or other personally identifiable information (PII) is written to the serial console, it is logged with all other visible text. It is highly recommended to use SSH to transmit sensitive data instead of using the serial console for this purpose.
```
Comment (gaurav-nelson) [inline virt/support/virt-troubleshooting.adoc:60]: **nitpick:** s/transmit/send

Comment (gaurav-nelson) [inline virt/support/virt-troubleshooting.adoc:60]: - **issue:** Insted of `It is highly recommended to use SSH ...` please use `Red Hat recommends using SSH ...`.
- **nitpick:** We can also remove `for this purpose` without changing the meaning.

**Comment by gaurav-nelson** [comment]
Thank you @ousleyp 
A few nitpicks from me; otherwise, it looks good.

**Comment by tiraboschi** [comment]
LGTM for me considering Gaurav suggestions.

File: modules/virt-enable-guest-log-default-cli.adoc (hunk 1 above)
Comment (ousleyp) [reply modules/virt-enable-guest-log-default-cli.adoc]: Haha, I usually do it on a per-module basis but it does look excessive with this many short modules in a row. Removing most instances. Thanks :)

**Comment by OhadRevah** [comment]
LGTM

**Review by OhadRevah — APPROVED** [review]
LGTM!

**Pull request merged**